code/modules/modular_computers/file_system/programs/card.dm

```diff
@@ -177,7 +177,7 @@
 		if("PRG_edit")
 			if(!computer || !authenticated || !target_id_card)
 				return
-			var/new_name = params["name"]
+			var/new_name = reject_bad_name(params["name"])		// if reject bad name fails, the edit will just not go through instead of discarding all input, as new_name would be blank.
```

Comment:
```suggestion
			var/new_name = reject_bad_name(params["name"]) // if reject bad name fails, the edit will just not go through instead of discarding all input, as new_name would be blank.
```